client/packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -92,56 +38,62 @@ const SyncSettingsForm = ({
   isDisabled: boolean;
   isSaving: boolean;
   isValid: boolean;
-  settings?: SyncSettings;
+  settings: SyncSettingsInput;
   onSave: () => void;
-  setSyncSettings: (syncSettings: SyncSettings) => void;
+  setSyncSettings: (syncSettings: SyncSettingsInput) => void;
 }) => {
   const t = useTranslation('common');
+  const setSettings = (
+    property: keyof SyncSettingsInput,
+    value: number | string
+  ) => setSyncSettings({ ...settings, [property]: value });
+
+  const { url, username, password, intervalSeconds } = settings;
   return (
     <form style={{ width: '100%' }}>
```

Comment:
What about adding this?

```
onKeyDown={e => {
        if (e.key === 'Enter') {
          onSave();
        }
      }}
```

as a keyboard user, I tried to submit by pressing Enter and it doesn't work 😢

Reply:
looks like adding type to button works: f125df49384081f23f131c62358996d96d6dd719

Reply:
weird. I tried that first and it didn't do anything for me. Was using firefox at the time, might be worth checking 🤷

Reply:
@andreievg  yeah - doesn't work in firefox. The whole page reloads the the settings are not saved. 
Works ok in chrome

Reply:
Hmm, oki added to carry over: https://github.com/openmsupply/open-msupply/issues/688
🙏 for testing.